disnake/ext/commands/bot_base.py

```diff
@@ -294,8 +293,7 @@ def check_commands(ctx):
                 return ctx.command.qualified_name in allowed_commands
 
         """
-        # T was used instead of Check to ensure the type matches on return
-        self.add_check(func)  # type: ignore
```

Comment:
The `type: ignore` and the accompanying comment are no longer necessary when using `CFT` instead of `T` - this change was a result of removing `T` entirely from this part of the code, though now that I think about it again, neither `T` nor `CFT` are entirely correct here. I'll revert this change and do another pass over the check/error methods in a future PR most likely.
ca46dde2f9a28c3de82149502da0ac8c9e7f9557